pkg/defaults/policies/files/external_exposure.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "78cd366c-4e06-4d9a-9b78-e547f18e7f0b",
+  "name": "Deployments with externally exposed endpoints",
+  "description": "Deployments with externally exposed endpoints represent a higher risk",
+  "rationale": "Deployments with services exposed outside the cluster are at risk of attempted intrusions because they are reachable outside of the cluster.",
+  "remediation": "Verify that service exposure outside of the cluster is required. If the service is only needed for Pod-to-Pod communication, use service type ClusterIP.",
```

Comment:
Not sure what the difference between pod-to-pod communication and intra-cluster communication is, because they seem the same? but doesn't need capitalization (whichever you use).

Reply:
or you could say "communication between pods" (or communication between clusters)

Reply:
Going with intra-cluster